lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -1,3 +1,4 @@
+include MiqAeYamlImportExportMixin
```

Comment:
This should not be included in the global namespace.

Note that I've seen this included globally elsewhere as well and those all need to be fixed.